src/db/gc/mod.rs

```diff
@@ -163,7 +163,10 @@ impl<DB: Blockstore + GarbageCollectable + Sync + Send + 'static> MarkAndSweep<D
     /// using CAR-backed storage with a snapshot, for implementation simplicity.
     pub async fn gc_loop(&mut self, interval: Duration) -> anyhow::Result<()> {
```

Comment:
```suggestion
    pub async fn gc_loop(&mut self, interval: Duration) {
```

Reply:
This won't work as the `services.spawn` expects a `Result`.